config/app.config

```diff
@@ -2,8 +2,8 @@
 // adjustable application runtime properties
 //---------------------------------------------
 BuildMajor = 2022
-BuildMinor = 2
-BuildRev = 1
+BuildMinor = 3
```

Comment:
3 was already used.  We need to bump to 2022.4.0 now.

Reply:
(It could also be 2022.3.1 if @robyww prefers.)

Reply:
I will make it 2022.4